packages.md

```diff
@@ -54,7 +78,58 @@ This can be achieved by using [`RENAME-PACKAGE`][rename-package]. For example:
 (re:scan "a" "abc")
 ~~~
 
+Please note that the original name (`:CL-PPCRE` in the example above) will
+**not** be available any more after renaming.
+
+## Package locks
+
+A package can be declared to be locked so that it cannot be modified by the
+user. Attempts to change its symbol table or redefine functions which its
```

Comment:
As a reader I ask myself: but why ? There's an answer in the SBCL doc:

> The package common-lisp and SBCL internal implementation packages are locked by default, including sb-ext. 

shall we (you :) ) add this explanation ?